index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
                 </ul>
 
             </div>
+                  <!-- start of work section -->
+
+<div class="work-container">
+    <div class="img-snapshot1">
+    <img src="images/Snapshoot1.png" alt="">
+    </div>
+    <div class="tonic1">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
```

Comment:
- [ ] Kindly make sure to center the cards to be aligned in the center of the screen to match the Figma design.
- [ ] Please make sure to change the background of the works section to match the Figma template. Also, please make sure that the background of the above section doesn't overlap with this section's background.
- [ ] [OPTIONAL] Kindly make sure to include the border-radius for the above section to match the Figma design.


| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/165791813-de5723df-00d1-445d-b8ea-53b6a3f2d90c.png) | ![image](https://user-images.githubusercontent.com/62290545/165791892-7128bf46-2fa3-4db9-8516-998fc707c8ee.png) |

Reply:
- [ ] Nice changing the background, like the previous reviewer said kindly make sure the cards are aligned in the center of the screen to match the Figma design.